src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs

```diff
@@ -65,6 +65,20 @@ internal static HashingEstimator Hash(this TransformsCatalog.ConversionTransform
             DataKind outputKind = ConvertDefaults.DefaultOutputKind)
             => new TypeConvertingEstimator(CatalogUtils.GetEnvironment(catalog), new[] { new TypeConvertingEstimator.ColumnOptions(outputColumnName, outputKind, inputColumnName) });
 
+        /// <summary>
+        /// Changes column type of the input columns.
+        /// </summary>
+        /// <param name="catalog">The conversion transform's catalog.</param>
+        /// <param name="columns">Specifies the names of the columns on which to apply the transformation.</param>
+        /// <param name="outputKind">The expected kind of the output column.</param>
+        public static TypeConvertingEstimator ConvertType(this TransformsCatalog.ConversionTransforms catalog,
+            InputOutputColumnPair[] columns,
```

Comment:
>columns [](start = 36, length = 7)

Don't you need to check for null for the columns arg to avoid null reference exception?

Reply:
I added the checks thanks for pointing out. I only fixed the extensions that are public. Not those that are internal.

---
In reply to: [268724517](https://github.com/dotnet/machinelearning/pull/3066#discussion_r268724517) [](ancestors = 268724517)